gemfirexd/tools/src/main/java/com/pivotal/gemfirexd/tools/GfxdDistributionLocator.java

```diff
@@ -247,6 +247,13 @@ protected void addToServerCommand(List<String> cmds,
     if (this.bindAddress == null) {
       this.bindAddress = FabricLocator.LOCATOR_DEFAULT_BIND_ADDRESS;
     }
+
+    Properties prop = (Properties) options.get(PROPERTIES);
+    if (prop.get(DistributionConfig.DISABLE_AUTO_RECONNECT_NAME) == null &&
+            prop.get(DistributionConfig.GEMFIRE_PREFIX + DistributionConfig.DISABLE_AUTO_RECONNECT_NAME) == null) {
+      prop.put(DistributionConfig.DISABLE_AUTO_RECONNECT_NAME, "false");
+    }
+
```

Comment:
For Property a more proper check is prop.getProperty because that also looks inside "defaults", though will not make a difference here I think.